packages/create-turbo/src/index.ts

```diff
@@ -162,26 +162,32 @@ async function run() {
   );
 
   // merge package.jsons
-  let appPkg = require(path.join(sharedTemplate, "package.json"));
+  let sharedPkg = require(path.join(sharedTemplate, "package.json"));
+  let projectPkg = require(path.join(projectDir, "package.json"));
 
-  // add current versions of remix deps
+  // add current versions of wildcard deps and merge
   ["dependencies", "devDependencies"].forEach((pkgKey) => {
-    for (let key in appPkg[pkgKey]) {
-      if (appPkg[pkgKey][key] === "*") {
-        appPkg[pkgKey][key] = `latest`;
+    // merge dependencies, giving priority to the project deps
+    sharedPkg[pkgKey] = {
```

Comment:
Merging deps allows us to specify common dependencies in `templates/_shared_ts/package.json`, and package manager specific overrides in `templates/{packageManager}/package.json`